windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -506,9 +508,115 @@ function TCompileKeymanWeb.JavaScript_ContextValue(fkp: PFILE_KEY; pwsz: PWideCh
     Result := Result + Format('",%d)', [Cur - StartQuotes]);
 end;
 
+// Used when targeting 10.0+, after the introduction of FullContextMatch/KFCM.
+function TCompileKeymanWeb.JavaScript_FullContextValue(fkp: PFILE_KEY; pwsz: PWideChar): string;
+
+  function IsRegExpSpecialChar(ch: WideChar): Boolean;
+  begin
+    Result := Pos('"\^$*+?{}.()|[]/', ch) > 0;
```

Comment:
Good to have a source reference comment for this. e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions has an example that includes the same set.

Reply:
Uh, okay... but that's copied straight from the parallel existing legacy function, so I guess I should add it there too?

Reply:
Righto. In that case, carry on.